scripts/cli.sh

```diff
@@ -16,7 +16,7 @@ write_to_file() {
     printf "# %s\n\n\`\`\`\n%s\n\`\`\`" "$program" "$cmd" > "$file"
 
     # Adjust the width of the help text and append to the end of file
-    sed -i -e '$a\'$'\n''\n''<style> .content main {max-width:88%;} </style>' "$file"
+    printf '%s\n\n' '' '<style> .content main {max-width:88%;} </style>' >> "$file"
```

Comment:
For consistency, may I suggest we use double quote "" (as in the code above)? We can also remove the `''` in the middle of the line for simplification.
The following should do it:
```suggestion
    printf "\n\n%s\n" "<style> .content main {max-width:88%;} </style>" >> "$file"
```
The `\n\n` is to create an empty line after the code block

Reply:
I've made the requested changes @chong-he